### PR TITLE
Make app compliant with sub-directory urls

### PR DESCRIPTION
**Pull request by alexarn**
Test plan:
  - Change your web server configuration to have a base url like http://localhost/subdir/
  - So that the index page is at http://localhost/subdir/index.php
  - Test all links :^)

### CODE REVIEW

File: templates/absences/add.html.twig [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% extends 'base.html.twig' %}
 
 {% block specificjs %}
-  <script type='text/JavaScript' src='{{ app.request.getBaseURL() }}/absences/js/modif.js?version={{ version }}'></script>
+  <script type='text/JavaScript' src='{{ app.request.getSchemeAndHttpHost() }}/absences/js/modif.js?version={{ version }}'></script>
 {% endblock %}
```
Comment (jeromecombes) [inline templates/absences/add.html.twig:7]: You should use "asset" : 
<pre>
<script type='text/JavaScript' src='{{ asset("absences/js/modif.js") }}?version={{ version }}'></script>
</pre>

**Review by jeromecombes — CHANGES_REQUESTED** [review]
Please use the asset function
e.g. : 
<pre><script type='text/JavaScript' src='{{ asset("absences/js/modif.js") }}?version={{ version }}'></script></pre>

File: public/conges/js/script.conges.js [hunk 2]
```diff
@@ -413,7 +413,7 @@ function verifConges(){
```
Comment (jeromecombes) [inline public/conges/js/script.conges.js:413]: line 411
<pre>
- baseURL = '';
+ baseURL = '/';
</pre>

**Review by jeromecombes — CHANGES_REQUESTED** [review]

**Review by jeromecombes — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**